README.md

```diff
@@ -74,13 +122,23 @@ Visit [http://localhost:3000/](http://localhost:3000/) to see the locally runnin
 ## Testing
 Make sure you have the same version of Node.js as specified in `package.json`, otherwise you may have trouble running the front end tests.
 
-Run tests with:
+If you are using Docker for development, run:
+```shell
+docker-compose run app npm test
+```
+
+If you are running Checklistomania locally, run:
 ```shell
 npm test
 ```
 
 Run eslint either via a code editor plugin (such as Atom's [linter-eslint](https://github.com/AtomLinter/linter-eslint)),
-or from the command line with:
+or from the command line. If you are using Docker, run:
+```shell
+docker-compose run app npm run eslint
```

Comment:
I was thinking another option here might be to add `/checklistomania/node_modules/.bin` to the `PATH` of the `app` container, so that one could just do `docker-compose run app eslint .`... However, if you ever change `npm run eslint` to do more than it currently does, then things wouldn't work out.

Still, adding `/checklistomania/node_modules/.bin` to the container's `PATH` could make things more convenient for anyone who wants to run `eslint` or other utilities directly, I suppose.